pkg/sypgp/sypgp.go

```diff
@@ -569,9 +569,36 @@ func SelectPrivKey(el openpgp.EntityList) (*openpgp.Entity, error) {
 	return el[n], nil
 }
 
-// SearchPubkey connects to a key server and searches for a specific key
-func SearchPubkey(search, keyserverURI, authToken string) error {
+// formatMROutput will take a machine readable input, and convert it to fit
+// on a 80x24 terminal.
+func formatMROutput(mrString string) string {
+	retList := "KEY ID    BITS  NAME/EMAIL\n\n"
+	count := 0
+
+	key := strings.Split(mrString, "\n")
+
+	for _, k := range key {
+		nk := strings.Split(k, ":")
```

Comment:
Can you leave some comments to this function. It's a bit hard to understand what this code do.

Reply:
Yes, I will also explain a bit here in more details; This function takes a machine readable input, that will look like:

```
info:1:5
pub:4E28E95609D65D3C5BEA9731F1E47D55A7F3A56C:1:4096:1540588176::
uid:westleyk (examples) <westley@sylabs.io>:1540588176::
pub:C7FA561F27F9CBEF7EF2C59CB7EE21014110093C:1:4096:1547537777::
uid:WestleyK (on-dell-xps) <westleyk@nym.hush.com>:1547537777::
pub:342E65CAA9C26D31A2B660AA029DF2F9691BA704:1:4096:1551486027::
uid:WestleyK (XPS) <westley@sylabs.io>:1551486027::
pub:E09D378BD534368E8BF2390386216BF0A30345A0:1:4096:1552506955::
uid:WestleyK (test key 1) <westley@sylabs.io>:1552506955::
pub:7605BC2716168DF057D6C600ACEEC62C8BD91BEE:1:4096:1560555332::
uid:WestleyK (Testing key; used for signing test containers) <westley@sylabs.io>:1560555332::
```

And convert it to look like:

```
Showing 5 results

KEY ID    BITS  NAME/EMAIL

A7F3A56C  4096  westleyk (examples) <westley@sylabs.io>

4110093C  4096  WestleyK (on-dell-xps) <westleyk@nym.hush.com>

691BA704  4096  WestleyK (XPS) <westley@sylabs.io>

A30345A0  4096  WestleyK (test key 1) <westley@sylabs.io>

8BD91BEE  4096  WestleyK (Testing key; used for signing test containers) <westley@sylabs.io>

```

I was really hoping theres a more elegant way to do this, like a external package, but I could not find any.